src/Polysemy.hs

```diff
@@ -10,8 +10,8 @@ module Polysemy
   , runM
 
   -- * Interoperating With Other Monads
-  , Lift (..)
-  , sendM
+  , Embed (..)
+  , embed
 
     -- * Lifting
```

Comment:
Perhaps this also need a better section name (?)

Reply:
I think it's ok! But keen eye!